client/my-sites/site-settings/amp/wpcom.jsx

```diff
@@ -104,5 +106,11 @@ class AmpWpcom extends Component {
 export default connect(
 	( state ) => ( {
 		siteSlug: getSelectedSiteSlug( state ),
-	} )
+	} ),
+	( dispatch ) => {
```

Comment:
There is a short syntax that automatically binds the action creators to `dispatch`, so

```
( dispatch ) => {
	const trackTracksEvent = ( name, props ) => dispatch( recordTracksEvent( name, props ) );
	return {
		trackTracksEvent,
	};
},
```

could be shortened to just:

```
{ recordTracksEvent }
```

Then we could also call it like this: `this.props.recordTracksEvent()` and avoid the tongue-twister prop name 😉